plugins/frontend/backstage-plugin-github-insights/src/mocks/scmIntegrationsApiMock.test.ts

```diff
@@ -45,3 +45,10 @@ export const createScmIntegrationsApiMock = (
     ]),
   );
 };
+
+
+describe('github-insights', () => {
```

Comment:
lol. I'll allow it.

Reply:
ah, linter doesn't. You can export a function that gets called and you can pass the offending dependency into that function from an actual test file.